packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -211,6 +211,40 @@ const EDiscovery = WebexPlugin.extend({
     });
   },
 
+  @waitForValue('@')
+  @oneFlight({keyFactory: (reportUrl, options) => createOneFlightKey(reportUrl, options)})
+  /**
+   * Retrieves content associated with a report
+   * @param {String} reportUrl - full destination address (including report id parameter) for the http request being sent
+   * e.g. 'http://ediscovery-intb.wbx2.com/ediscovery/api/v1/reports/3b10e625-2bd5-4efa-b866-58d6c93c505c'
+   * @param {Object} options - optional parameters for this method
+   * @param {number} options.offset - start position from which to retrieve records
+   * @param {number} options.size - the number of records to retrieve
+   * @param {number} options.timeoutMs - connection timeout in milliseconds, defaults to 30s
+   * @returns {Promise<ResponseEntity<[Activity]>>} Http response containing the activities
+   */
+  getRemoteContent(reportUrl, options) {
+    if (!reportUrl) {
+      throw Error('Undefined parameter');
+    }
+
+    const reportId = this._getReportIdFromUrl(reportUrl);
+
+    if (!reportId) {
+      throw Error('Report url does not contain a report id');
+    }
+
+    // use spread operator to set default options
+    options = {...this.config.defaultOptions, ...options};
+
+    return this.request({
+      method: 'GET',
+      url: `${reportUrl}/contents`,
+      qs: {offset: options.offset, size: options.size},
+      timeout: options.timeoutMs
+    });
+  },
+
```

Comment:
To work with ediscovery reports with content in multiple federated environments we may need to make requests outside the home cluster to retrieve that content, these APIs allow us to achieve that by setting the url property on teh request parameters, rather than the service and resource we previously used.